src/views/migrationpolicies/list/components/MigrationPolicyCreateForm/copmonents/SelectorLabelMatchGroup/SelectorLabelMatchGroup.scss

```diff
@@ -0,0 +1,14 @@
+.mp-match-label-group {
+    margin-top: var(--pf-global--spacer--xs);
+    padding-top: var(--pf-c-label-group--m-category--PaddingTop);
+    padding-right: var(--pf-c-label-group--m-category--PaddingRight);
+    padding-bottom: var(--pf-c-label-group--m-category--PaddingBottom);
+    padding-left: var(--pf-c-label-group--m-category--PaddingLeft);
+    border: var(--pf-c-label-group--m-category--BorderWidth) solid
+      var(--pf-c-label-group--m-category--BorderColor);
+}
+
+.kv-migration-policy__label-vm {
+    color: var(--pf-global--link--Color);
+}
+  
```

Comment:
I can see here on github for this line "No newline at end of line". Maybe a newline missing or so? Can you check, please? :)

Reply:
nice catch fixed :)